web-animations-2/Overview.bs

```diff
@@ -2443,6 +2471,10 @@ Add:
 >      <a href="#dom-keyframeanimationoptions-rangeend">rangeEnd</a>.
 >     When reading the attribute, the returned value is either a
 >     {{TimelineRangeOffset}} or the {{DOMString}} "normal".
+>
+> :  <dfn attribute for=Animation>progress</dfn>
+> ::  Specifies the progress of the <a>animation</a> as a proportion of its
+>     [=animation/associated effect=]'s [=end time=].
```

Comment:
This should link to the algorithm that defines how to calculate the progress.

Reply:
Done.